frontend/src/styles/bootstrap.scss

```diff
@@ -19,7 +19,7 @@
 @import '../../node_modules/bootstrap/scss/images';
 @import '../../node_modules/bootstrap/scss/containers';
 @import '../../node_modules/bootstrap/scss/grid';
-// @import '../../node_modules/bootstrap/scss/tables';
+@import '../../node_modules/bootstrap/scss/tables';
 // @import '../../node_modules/bootstrap/scss/forms';
 @import '../../node_modules/bootstrap/scss/buttons';
```

Comment:
For components, we should use angular-material components because they are better integrated in the angular framework in regard of performance and so on.
They can be styled differently if we do not want the default styles.

Reply:
So i would suggest removing the buttons, because we use angular material buttons.

Reply:
Removed the import